docs/api/Actions.md

```diff
@@ -0,0 +1,57 @@
+# API: Actions
+
+Actions in Fluxible are stateless functions that receive three parameters:
+
+ * [`actionContext`](#Action Context) - Provides access to Flux methods
+ * `payload` - The action payload
+ * `done` - A function to be called when the action has been completed
+
+```js
+module.exports = function myAction(actionContext, payload, done) {
+    setTimeout(function () { // simulate async
+        actionContext.dispatch('MY_ACTION', payload);
+        done();
+    }, 10);
+};
+```
+
+The first action for a context is called via `context.executeAction(myAction, payload)` but actions can also be fired by other actions:
```

Comment:
This is discussed in the FluxibleContext.md. I think I will reword this line though.